src/ui/input/radio_group.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+export default class RadioGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      options: [],
+    };
+  }
+
+  componentDidMount() {
+    const { name } = this.props;
+    // for options need create a global variable like a:
+    // window.{name}_options = [value: 'test', label:'test']
+    // where {name} its name of field
+    let isInvalidOptions = false;
+    const options = window[`${name}_options`];
+    // resolve options
+    const result = Array.isArray(options) ? options : [];
+    result.forEach((item) => {
+      if (typeof item === 'object' && item.hasOwnProperty('value') && item.hasOwnProperty('label')
+      && typeof item.value === 'string' && typeof item.label === 'string') {
+        isInvalidOptions = isInvalidOptions || false
+      } else {
+        isInvalidOptions = true;
+      }
+    })
+    if (isInvalidOptions) {
+      console.warn(`Invalid options for ${name} field`)
```

Comment:
Do we need this warning? Is it only for debugging?

Reply:
we need this warning